structures.c

```diff
@@ -0,0 +1,61 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+
+typedef struct Date {
+    int day, month, year;
+} Date;
+
+
+typedef struct Person {
+    char name[], surname[];
+    char sex[];
+    Date birthday;
+} Person;
+
+
+typedef struct Product {
+    char product_name[];
+    double price;
+} Product;
+
+
+typedef struct Equipment {
+    char equipment_name[];
+    Product product;
+} Equipment;
+
+
+typedef struct Profession {
+    char profession_name[];
+    double wage_rate;
+    Equipment equipment;
+} Profession;
+
+
+typedef struct Employee {
+    struct Person;
+    struct Profession;
+    double salary;
+    Date employed;
+    Profession profession;
+} Employee;
+

```

Comment:
If Yuor struct contains reference to another struct or this another struct it can be analgue to inheritence